ext/kubernetes/deployment/init.go

```diff
@@ -96,6 +97,27 @@ func initiateDeploymentProcedure(d *Deployment) {
 				deploySuccess,
 				strings.Join(d.GetPods(), " "),
 			)
+			if strings.Contains(componentName, "prod") && strings.Contains(clusterName, "eu") {
```

Comment:
I would suggest to not hardcode this here. Instead, would definitely better to create a configuration entry for conditions or even a template that would give an specific instruction to Signa.